file-uploader.js

```diff
@@ -47,8 +47,11 @@ export function fileUploader(submissionBody, submissionFormat, requestHeaders, b
   };
 
   return fileUploaderUtil.validateSubmission(submissionBody, submissionFormat, baseOptions)
-    .then((validSubmission) => {
-      validatedSubmission = validSubmission;
+    .then((_validSubmission) => {
+      // QPPSF-5596, part of the validation logic for NonProportion measures for PY 2019, is to add a new field during validation,
+      // to prevent additional fields being added, we are going to send the original submission object
+      // to the api since the validation adds some properties
+      validatedSubmission = JSON.parse(submissionBody);
```

Comment:
Is there a point in calling `fileUploaderUtil.validateSubmission` any more if the result is not used? Could we just replace that call with `return fileUploaderUtil.getExistingSubmission(JSON.parse(submissionBody), baseOptions)` instead?

idk what the validation logic did before, but is it correct to assume basically we are offloading validation to the submission api?? or is there something else that the `validateSubmission` function does that we need?

Reply:
all good questions, I have no clue

Reply:
We need the JSON.parse here because in `getExistingSubmission` method we do something like `submission.nationalProviderIdentifier`and the fact that submission is string when passed into `validateSubmission` we need to parse it so the properties can be read from it

Does my comment makes sense?

Reply:
@joshboley , do you know if we need anything from the returned `validateSubmission` object?  Or is it okay that we continue to use the `submissionBody` so long as it passed validation?

Reply:
No, that's the point of this PR...don't use the submission returned from the validate submission endpoint.  It will have extra fields that will break your subsequent POST

Reply:
Right, just making sure that that included everything.  Thanks :)